index.html

```diff
@@ -240,16 +240,34 @@ <h3 class="title has-subtitle">Already works with your API</h3>
         React-admin supports most REST and GraphQL dialects, and writing your
         own adapter takes minutes.
       </p>
-      <div class="grid">
-        <img src="assets/adapters/openapi.png" alt="Open API" />
-        <img src="assets/adapters/jsonapi.png" alt="Json API" />
-        <img src="assets/adapters/graphql.png" alt="GraphQL" />
-        <img src="assets/adapters/apiplatform.png" alt="API Platform" />
-        <img src="assets/adapters/django.png" alt="Django" />
-        <img src="assets/adapters/firebase.png" alt="Firebase" />
-        <img src="assets/adapters/hasura.png" alt="Hasura" />
-        <img src="assets/adapters/prisma.png" alt="Prisma" />
-        <img src="assets/adapters/spring.png" alt="Spring" />
+      <div class="grid clickable">
+        <a target="openapi" href="https://www.openapis.org/">
```

Comment:
as much as possible, you should link to the dataProvider for that protocol, not to the protocol spec...

Reply:
not founded for openapi ;(
but changed for the others